tests/e2e/tests/plugins/VscodeShellcheckPlugin.spec.ts

```diff
@@ -52,11 +48,9 @@ suite(`The 'VscodeShellcheckPlugin' test`, async () => {
             await browserTabsUtil.navigateTo(factoryUrl);
         });
 
-        test('Wait until created workspace is started', async () => {
-            await ide.waitAndSwitchToIdeFrame();
-            workspaceName = await workspaceNameHandler.getNameFromUrl();
-            CheReporter.registerRunningWorkspace(workspaceName);
+        workspaceHandlingTests.obtainWorkspaceNameFromStartingPage();
```

Comment:
It's strange to see code between the tests.

IMHO, it's part of `Wait until created workspace is started` test.